documentation/book/ref-metrics-alertmanager-examples.adoc

```diff
@@ -0,0 +1,48 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-metrics-prometheus-alertmanager.adoc
+[id='ref-metrics-alertmanager-examples-{context}']
+
+= Alerting rule examples
+
+Example alerting rules for Kafka and Zookeeper metrics are provided with {ProductName} for use in a xref:proc-metrics-deploying-prometheus-{context}[Prometheus deployment].
+
+The rules are defined in a `PrometheusRule` resource in the link:https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus-rules.yaml[`prometheus-rules.yaml`^] file
+
+.Kafka alerting rules
+
+* `UnderReplicatedPartitions`: gives the number of partitions for which the current broker is the lead replica but the follower replicas are behind.
+This metric provides insights about offline brokers that host the follower replicas.
+An alert is raised when this value is greater than zero, providing information on the under-replicated partitions for each broker.
+
+* `AbnormalControllerState`: indicates whether the current broker is the controller for the cluster.
+The metric can be 0 or 1.
+During the life of a cluster, only one broker should be the controller and the cluster always needs to have an active controller.
+Having two or more brokers saying that they are controllers indicates a problem.
+An alert is raised when the sum of all the values for this metric on all brokers is not equal to 1, meaning that there is no active controller (the sum is 0) or more than one controller (the sum is greater than 1).
+
+* `UnderMinIsrPartitionCount`: indicates that the minimum number of in-sync replicas (ISRs) for a lead Kafka broker, specified using `min.insync.replicas`, that must acknowledge a write operation has not been reached.
+The metric defines the number of partitions that the broker leads for which the in-sync replicas count is less than the minimum in-sync.
+An alert is raised when this value is greater than zero, providing information on the partition count for each broker that did not achieve the minimum number of acknowledgments.
+
+* `OfflineLogDirectoryCount`: indicates the number of log directories which are offline (for example, due to a hardware failure) so that the broker cannot store incoming messages anymore.
+An alert is raised when this value is greater than zero, providing information on the number of offline log directories for each broker.
+
+* `KafkaRunningOutOfSpace`: indicates the remaining amount of disk space that can be used for writing data.
+An alert is raised when this value is lower than 5GiB, providing information on the disk that is running out of space for each persistent volume claim.
```

Comment:
5GiB doesn't sound like very much. How can the user configure this?

Reply:
@ppatierno or @stanlyDoge - can we check on this alert?

Reply:
User can redeploy `prometheus-rules.yaml` file with his own values. What value should be set as "default"?

Reply:
I've added that teh threshold value can be changed in `prometheus-rules.yaml`